pkg/plugins/policies/meshmetric/api/v1alpha1/validator.go

```diff
@@ -77,6 +79,12 @@ func validateBackend(backends *[]Backend) validators.ValidationError {
 			} else {
 				verr.Add(validators.ValidatePort(path.Field("port"), backend.Prometheus.Port))
 			}
+		case OpenTelemetryBackendType:
+			if backend.OpenTelemetry == nil {
+				verr.AddViolationAt(path.Field("openTelemetry"), validators.MustBeDefined)
+			} else if !govalidator.IsURL(backend.OpenTelemetry.Endpoint) {
```

Comment:
just wanted to check if we have the same validation for MAL and there we only check if it's defined - https://github.com/kumahq/kuma/blob/fecb3cf9cbb4d2552d8b24aa676ef1446095d036/pkg/plugins/policies/meshaccesslog/api/v1alpha1/validator.go#L116-L120 @lobkovilya - any reason for that? I think checking `govalidator.IsURL` is safer.

Reply:
I've created issue for this: https://github.com/kumahq/kuma/issues/8884